searchcore/src/vespa/searchcore/proton/server/threading_service_config.cpp

```diff
@@ -22,7 +22,7 @@ namespace {
 uint32_t
 calculateIndexingThreads(uint32_t cfgIndexingThreads, double concurrency, const HwInfo::Cpu &cpuInfo)
 {
-    double scaledCores = cpuInfo.cores() * concurrency;
+    double scaledCores = std::min(12.0, cpuInfo.cores() * concurrency);
```

Comment:
Please add a description on why we do the capping. Otherwise ok.

Reply:
Fixed